lib/features/booking/add_inspection_booking_screen.dart

```diff
@@ -94,7 +94,9 @@ class _AddInspectionBookingBody extends State<AddInspectionBookingBody> {
                 ),
                 context: context,
                 builder: (_) {
-                  return const UpdatePhoneNoBottomSheetView();
```

Comment:
why do u need to add a `BlocProvider` here? `AddInspectionBookingCubit` is created in the same screen (line 28 above), you can just reuse it here without providing a new cubit